feature/system/management/README.md

```diff
@@ -0,0 +1,146 @@
+# MGT-1: Management HA solution test
+
+## Summary
+
+- Test management HA
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_4.testbed
+
+## Procedure
+
+### Applying configuration
+
+For each section of configuration below, prepare a gnmi.SetBatch with all the configuration items appended to one SetBatch. Then apply the configuration to the DUT in one gnmi.Set using the `replace` option
+
+### Initial Setup:
+
+*   Connect DUT port-1, 2, 3 and 4 to ATE port-1, 2, 3 and 4
+*   Create VRF "mgmt" on DUT
+    *   /network-instances/network-instance[name=mgmt]/vrfs/vrf[name=mgmt]/config/name = mgmt
+    *   /network-instances/network-instance[name=mgmt]/vrfs/vrf[name=mgmt]/config/route-distinguisher = 64512:100
+*   Create an IPv6 networks ```ateNet``` attached to ATE port-1, 2, 3 and 4
+*   Create a loopback interface "lo0" on DUT and assign it an IPv6 address
+    *   /interfaces/interface[name=lo1]/config/name = lo1
+    *   /interfaces/interface[name=lo1]/config/type = softwareLoopback
+    *   /interfaces/interface[name=lo1]/subinterfaces/subinterface[index=0]/ipv6/addresses/address/config/ip
+    *   /interfaces/interface[name=lo1]/subinterfaces/subinterface[index=0]/ipv6/addresses/address/config/prefix-length
+*   Configure the loopback interface to participate in the VRF "mgmt"
+    *   /interfaces/interface[name=lo1]/subinterfaces/subinterface[index=0]/ipv6/vrf/name = mgmt
```

Comment:
I don't think `/ipv6/vrf` OC path is legit? (Same apply to the following OC path that put interface to a VRF)
https://openconfig.net/projects/models/schemadocs/jstree/openconfig-interfaces.html

cc: @rszarecki

Reply:
Fixed